### ISSUE TITLE
Formatting information not reported in Excel with UIA

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14954]
### Steps to reproduce:

1) Open Excel
2) Ensure "colors" is enabled in NVDA's document formatting settings (NVDA+control+d)
3) Ensure Use UI Automation to access Microsoft Excel controls is enabled in NVDA's advanced formatting.
4) Set a cell to have different colours (alt+h, h, then select a background colour, or alt+h, f, c, then select a font colour).  Or format as a table and it will auto colour alternate rows light blue aqua
5) Navigate to and from the cell with different formatting.
6) Press NVDA+f to report the formatting.
7) Turn off using UIA in Excel and repeat.

### Actual behavior:

No colour formatting is reported when navigating in Excel with UIA enabled.
Colour formatting IS reported when requested with NVDA+f.
Colour formatting is also reported when navigating in Excel with UIA disabled.

### Expected behavior:

Colour formatting should be reported if "colors" is enabled in NVDA's document formatting settings.

#### NVDA version:

2023.1 installed

#### Windows version:
Windows 11 (64-bit) Version: 22H2, Build: 22621.1702 
#### Name and version of other software in use when reproducing the issue:
Office 365 (64-bit) Version: 16.0.16327.20248 
#### If NVDA add-ons are disabled, is your problem still occurring?
No add-ons

**Comment by Qchristensen** [comment]
@dkager Is that any different on empty cells vs on cells with content?  With my steps above, I get the same results regardless of whether cells have text or not.  IE I don't get any colour formatting information when navigating with UIA enabled - regardless of whether the cell has contents or not.

**Comment by dkager** [comment]
What I did:
1. Open Excel 365.
2. Press NVDA+f.
> No formatting information
3. Put a single space (or pretty much any other text) in the cell.
4. Press NVDA+f.
> style Normal Calibri 11.0 pt black on white align left

Incidentally, arrowing onto a cell containing just a space gives me no output at all using NVDA right now, but that's a story for a different issue. :-)

**Comment by Qchristensen** [comment]
My apologies - it is different you are right - I was trying with UIA on and off to see the difference, and with UIA enabled NVDA doesn't report any information on completely blank cells with NVDA+f, but if you have even just a space, it does, as you found.
